rootfs/standard/usr/bin/mynode_post_upgrade.sh

```diff
@@ -20,6 +20,14 @@ apt -y install libatlas-base-dev libffi-dev libssl-dev glances python3-bottle
 apt -y -qq install apt-transport-https ca-certificates
 apt -y install libgmp-dev automake libtool libltdl-dev libltdl7
 
+# Install Whirlpool
+apt -y install openjdk-8-jre
+sudo -u bitcoin mkdir -p /opt/mynode/whirlpool
+cd /opt/mynode/whirlpool
+rm -rf *.jar
+sudo -u bitcoin wget https://github.com/Samourai-Wallet/whirlpool-runtimes/releases/download/cli-0.9.1/whirlpool-client-cli-0.9.1-run.jar
```

Comment:
if you use WHIRLPOOL_VERSION variable, it would be easier to implement upgrades in the future. Also, rename the jar file to something simple, so that the setup_whirlpool script doesn't need to know the version.

Reply:
East fix.

Reply:
I reformatted the install script to something similar to that of the other installs/upgrades